gapis/trace/android/validate/cc/cube.h

```diff
@@ -0,0 +1,154 @@
+#pragma once
```

Comment:
Missing copyright notice. This is probably from Vulkan SDK, and if so should be under `third_party`